lib/core/src/sdk.rs

```diff
@@ -3360,7 +3362,7 @@ impl LiquidSdk {
                 .insert_or_update_payment_details(PaymentTxDetails {
                     tx_id,
                     destination,
-                    description: prepare_response.comment.clone(),
+                    description,
```

Comment:
What exactly do we consider as description for lnurlpay payments? I can see several options but just want to be accurate at what we want. Also is it aligned with the description we show in greenlight sdk?

Reply:
On app level, we show `text/long-desc` if it's available and fallback to mandatory `text/plain` field on `metadata_str` for `description`.

Currently, both SDK's do not persist this value as `description` for LNURLp payments.

Reply:
@erdemyerebasmaz  I see. Here we only take the text/plain ignoring the text/long-desc is that OK?

Reply:
Why not taking the description from payment.description? It should contain the invoice description shouldn't it?